base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -4326,3 +4607,127 @@ fn get_correct_accumulated_difficulty() -> Vec<(u64, U512)> {
     #[cfg(not(any(tari_target_network_mainnet, tari_target_network_nextnet)))]
     vec![]
 }
+
+// This function will read and verify all metadata keys in the metadata db can be read, or delete them otherwise.
+fn verify_metadata_keys(db: &LMDBDatabase) -> Result<(), ChainStorageError> {
```

Comment:
the way this is written is a bad idea. If say the chain_height is corrupted, then other migrations wont work correctly, and just deleting it wont fix anything. If any if these keys are broken, we can shutdown the node, and delete the db. These keys need to match what is written in the db, we cant just delete them.